boefjes/boefjes/plugins/kat_nmap/normalize.py

```diff
@@ -44,9 +44,7 @@ def get_ip_ports_and_service(host: NmapHost, network: Network, netblock: Referen
             yield ip_port
 
             service_name = service.service
-            if port == 80:
-                service_name = "http"
-            if port == 443:
+            if service.tunnel == "ssl":
```

Comment:
Are we convinced that SSL always means https? As many other services could also be carried over SSL/TLS.
Nmap itself seems to return ssl/https (possibly with an extra ? at the end if its not sure).

Reply:
Good point. Addressed in cd27d89a1cc982065fbb467ee1b35401850462f7.